app/controllers/rubcop_test_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RubocopController  < ApplicationController
+	def initialize
+		print "this is so bad patern"
+	end
+
+	def print_fib(range)
+		range.each do |n|
```

Comment:
__[staging][rubocop]__
* Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Use 2 (not 3) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->